packages/react-components/src/components/CaptionsSettingModal.tsx

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
I have a feeling that this is more like a component should living in composite folder, we can move this back if we need it to be exposed

Reply:
There is a captions setting modal component in the composite folder as well, acting as a wrapper and pulling required data using use props for. 
Feel like it can be valuable as a component for customers? for now it's marked internal so it's not exposed, I also have an internal storybook page for this

Reply:
Since this is internal, I wouldn't be too worried about this, let's leave it as it is